source/blog/2017-03-29-migrate-fedora-atomic-host-to-overlay2.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Migrating the Docker Storage Driver to overlay2
+author: miabbott
+date: 2017-03-29 16:00:00 UTC
+published: true
+comments: true
+tags: atomic, fedora, docker, overlay2
+---
+
+On the [Project Atomic mailing list](https://lists.projectatomic.io/projectatomic-archives/atomic/2017-March/msg00015.html), [Colin Walters](https://twitter.com/cgwalters) posted
+a quick set of instructions on how to migrate the Docker storage driver from `devicemapper` to `overlay2` on Fedora Atomic Host.
+
+The `overlay2` driver will be the default storage driver in Fedora 26, so making this change will give you a chance to test out your workloads
+before the Fedora 26 release.
```

Comment:
Might be worth hotlinking or just adding a link here to the corresponding proposal: https://fedoraproject.org/wiki/Changes/DockerOverlay2.